images/linux/scripts/installers/homebrew.sh

```diff
@@ -11,14 +11,13 @@ source $HELPER_SCRIPTS/install.sh
 
 # Install the Homebrew on Linux
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
-eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
+brew_shellenv="/home/linuxbrew/.linuxbrew/bin/brew shellenv"
 
-# Update /etc/environemnt
-## Put HOMEBREW_* variables
-## The variable `HOMEBREW_SHELLENV_PREFIX` will be exported to avoid adding duplicate entries to the environment variables.
-HOMEBREW_SHELLENV_PREFIX='' brew shellenv | grep 'export HOMEBREW'| sed -E 's/^export (.*);$/\1/' | sudo tee -a /etc/environment
+# Update /etc/environment
```

Comment:
What about adding `eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)` directly to `/etc/environment`?

Reply:
> What about adding `eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)` directly to `/etc/environment`?

Scripting or glob expansion cannot be used with /etc/environment. The file only accepts variable=value pairs.

Reply:
Gotcha 👍🏻